kalite/topic_tools/__init__.py

```diff
@@ -430,7 +430,7 @@ def get_exercise_data(request, exercise_id=None):
     if not exercise:
         return None
 
-    exercise_root = os.path.join(settings.KHAN_EXERCISES_DIRPATH, "exercises")
+    exercise_root = os.path.join(settings.KHAN_EXERCISES_DIRPATH)
```

Comment:
How? The edit to i18n above will make language pack exercise templates unpack into static/perseus/ke/exercises/{ka_lang_code} folder